onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -1222,6 +1222,26 @@ MlasQuantizeLinear(
     OutputType ZeroPoint
     );
 
+void
+MLASCALL
+MlasQuantizeLinearU4(
```

Comment:
there is no a similar dequantization implementation?

Reply:
No, dequantization for CPU EP DQ op does not currently use MLAS code.